examples/storage/couchdb/couchdb-petset.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: "apps/v1alpha1"
+kind: PetSet
+metadata:
+  name: kubetest001
+  namespace: default
+spec:
+  serviceName: kubetest001-backend
+  replicas: 3
+  template:
+    metadata:
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+        pod.alpha.kubernetes.io/init-containers: '[
+            {
+            "name": "configer-couch",
+            "image": "glyndwr/kube-init-couchdb",
+            "command": ["/bin/bash", "-c", "/./configer_couch.sh"],
+            "volumeMounts": [
+                {
+                    "name": "core-config",
+                    "mountPath": "/core-config/"
+                                  },
+                {
+                    "name": "pod-config-store",
+                    "mountPath": "/opt/couchdb/etc/"
+                                  }
+                              ],
+            "env": [
+                {
+                    "name": "INSTALL_LOCATION",
+                    "value": "/opt/couchdb"
+                }
+                              ] 
+              }
+            ]'
+      labels:
+        cluster: kubetest001
+        tier: backend
+        app: couchdb
+    spec:
+      containers:
+      - env:
+        - name: POD_IP
```

Comment:
Correct me if I'm wrong, but all you really need from that FQDN is the namespace and the name of the service to make the appropriate SRV query, right? I believe those are already available in the pod environment, I think using them directly would be cleaner than using the POD_IP to extract them from the FQDN.